src/shared/components/palette/organisation/write-organisation-field.component.html

```diff
@@ -54,7 +54,7 @@ <h2 class="heading-h2">Organisation name and address</h2>
           </div>
         </div>
       </div>
-      <div *ngIf="(selectedOrg$ | async).organisationIdentifier">
+      <div *ngIf="(selectedOrg$ | async)?.organisationIdentifier">
```

Comment:
_Suggestion:_
You could wrap this in a template and set line 37 to have an else:
`<div *ngIf="!(selectedOrg$ | async)?.organisationIdentifier; else selectedOrganisation" ...>`

And then:
```
<ng-template #selectedOrganisation>
  <table id="organisation-selected-table" ...>
    ...
  </table>
</ng-template>
```

Reply:
took your advice, thanks.